src/providers/ldap/sdap_async.c

```diff
@@ -2583,23 +2582,21 @@ struct sdap_posix_check_state {
     struct tevent_context *ev;
     struct sdap_options *opts;
     struct sdap_handle *sh;
-    struct sdap_search_base **search_bases;
     int timeout;
 
     const char **attrs;
     const char *filter;
-    size_t base_iter;
 
     bool has_posix;
 };
 
 struct tevent_req *
 sdap_posix_check_send(TALLOC_CTX *memctx, struct tevent_context *ev,
```

Comment:
I found in https://technet.microsoft.com/en-us/library/cc755809(v=ws.10).aspx in the 'Global catalog search base' that searches with a empty base will fail on the LDAP port. So I think it would be good to add a comment that this call should be only used with the GC maybe even rename it to sdap_gc_posix_check?